src/Tree/Entity/Repository/ClosureTreeRepository.php

```diff
@@ -150,7 +150,11 @@ public function childrenQueryBuilder($node = null, $direct = false, $sortByField
         }
 
         if ($sortByField) {
-            if ($meta->hasField($sortByField) && in_array(strtolower($direction), ['asc', 'desc'], true)) {
+            if (is_array($sortByField)) {
+                foreach ($sortByField as $key => $field) {
+                    $qb->addOrderBy('node.'.$field, is_array($direction) ? $direction[$key] : $direction);
```

Comment:
```suggestion
                    $qb->addOrderBy('node.'.$field, $direction[$key] ?? $direction);
```
Maybe? to handle possible array size missmatch

Reply:
It wouldn't work exactly like that, because if `$direction` is a string, `$direction[$key]` would return its first letter.